SCREENSHOTS.md

```diff
@@ -78,3 +78,44 @@ Current implementation of the utility is based on following external libraries:
 - [pngjs](https://github.com/lukeapage/pngjs): For image read/write to disk
 
 
+# Screenshot reporting
+
+This utility allows generating screenshot based reporting. For now, it has been created for helping on tests troubleshooting, although it could be extended and used also for formalizing execution evidences. 
+
+Screenshot reporting configuration:
+
+- setBasePath(string): void: Sets the base path where the screenshots will be located.
+- beforeSuite(suite: any): void: resets a spec (it) counter to zero. Intended to be called from beforeSuite hook.  
+- afterTest(test, context, status): Promise void: takes a screenshot and saves it for now in a basic folder structure. Intended to be called from the afterTest hook.
+  Structure example:
+  ````
+  - <"describe description string of 1st e2e-spec.ts file">/
+    - 01-<1st "it" expectation string of e2e test">
+    - 02-<2nd "it" expectation string of e2e test">
+  
+  - <"describe description string of 2nd e2e-spec.ts file">/
+    - 01-<1st "it" expectation string of e2e test">
+    - 02-<2nd "it" expectation string of e2e test">
+  ````
+### Usage: 
+
+Set the base path for the output of screenshot reporting. To be called from wdio.conf.ts beforeSession hook:
+```typescript
+beforeSession: function (config, capabilities, specs) {
+  ScreenshotReporter.setBasePath('../reports/screenshots/e2e');
+},
+```
+
+Call ScreenshotReporter.beforeSuite() method. To be called from wdio.conf.ts beforeSuite hook:
```

Comment:
I would surround method names on the text with backticks, so they are highlighed when the markdown is shown on read mode. For instance,  `ScreenshotReporter.beforeSuite()` is more readable than just ScreenshotReporter.beforeSuite(). I would apply backticks also to wdio.conf.ts file.